src/backends/plonky2/mainpod/operation.rs

```diff
@@ -34,6 +34,7 @@ impl OperationArg {
 pub enum OperationAux {
     None,
     MerkleProofIndex(usize),
+    MerkleTreeStateTransitionProofIndex(usize),
```

Comment:
Could you move this entry one position down so that it matches the order of the table?
https://github.com/0xPARC/pod2/blob/69e5c41b7f23a92a428556a5be3f2582a19183b4/src/backends/plonky2/circuits/mainpod.rs#L191-L197